.github/workflows/ci.yml

```diff
@@ -325,7 +318,7 @@ jobs:
           SLACK_HOOK: ${{ secrets.LINK_CHECKER_SLACK_HOOK }}
         run: |
           cd docs
-          poetry run linkchecker --config linkcheckerrc https://docs.webknossos.org > link_status || \
+          uv run linkchecker --config linkcheckerrc https://docs.webknossos.org > link_status || \
```

Comment:
May be put a `--frozen` here (repository is checked out: dependencies may update)